airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -52,12 +51,6 @@
 # Elasticsearch hosted log type
 EsLogMsgType = List[Tuple[str, str]]
 
-# Compatibility: Airflow 2.3.3 and up uses this method, which accesses the
-# LogTemplate model to record the log ID template used. If this function does
-# not exist, the task handler should use the log_id_template attribute instead.
-USE_PER_RUN_LOG_ID = hasattr(DagRun, "get_log_template")
```

Comment:
@eladkal @uranusjr do we need to consider this removal "breaking"?

Reply:
Arguably we should have made the constant private (with a leading underscore). Technically this is breaking, but since this is undocumented and I don’t see how it being useful to any user in the first place, so the removal is probably fine.